test/pluggy_elixir/auth_test.exs

```diff
@@ -0,0 +1,66 @@
+defmodule PluggyElixir.AuthTest do
+  use PluggyElixir.Case, async: false
+
+  alias PluggyElixir.Auth
+  alias PluggyElixir.Error.Unauthorized
+
+  setup do
+    config = Application.get_all_env(:pluggy_elixir)
+
+    on_exit(fn ->
+      Application.put_all_env(pluggy_elixir: config)
+    end)
+
+    :ok
+  end
+
+  describe "create_api_key/0" do
+    test "return a api_key", %{bypass: bypass} do
+      bypass_expect(bypass, "POST", "/auth", fn conn ->
+        assert %{
+                 "clientId" => _client_id,
+                 "clientSecret" => _client_secret,
+                 "nonExpiring" => _expiring
```

Comment:
```suggestion
                 "nonExpiring" => false
```

That is the default value right?

Reply:
Yes, but for that test I dont care about the values, just the body struct.